### PR TITLE
Update repository version

### PR DESCRIPTION
**Pull request by jcvenegas**
Add script that will help to bump versions for all the projects.

Fixes: #49

### CODE REVIEW

File: release/update-repository-version_test.sh [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+echo "${out}" | grep "no new version">> /dev/null
+OK
+
+echo "help option"
+out=$("${script_dir}/update-repository-version.sh" -h)
+output_should_contain "${out}" "Usage"
+OK
+
+echo "Local update version update should work"
+new_version=50.0.0
```
Comment (jodh-intel) [inline release/update-repository-version_test.sh]: I guess we won't hit this for a while :)

**Comment by jodh-intel** [comment]
lgtm

@klynnrif, @intelkevinputnam - ptal.

**Comment by jcvenegas** [comment]
@klynnrif , my bad, I closed the previous PR https://github.com/kata-containers/packaging/pull/77 and seems that I am not able to re-open again because I changed the branch. This include the previous fixes you suggested.

File: release/README.md [hunk 2]
```diff
@@ -2,20 +2,49 @@
...
+
+```bash
+$ ./update-repository-version.sh -h
+```
+
+### Update Kata projects to a new version ###
+Kata Containers is divided into multiple projects. With each release, all
+project versions are updated to keep the version consistent.
+
+To to update all versions for all projects, use the following:
```
Comment (klynnrif) [inline release/README.md]: Duplicate "to": To update all versions for all projects, use the following:

File: release/README.md [hunk 3]
```diff
@@ -2,20 +2,49 @@
...
+
+```bash
+$ make bump-kata-version NEW_VERSION=<new-version>
+```
+
+The makefile target bump-kata-version creates a GitHub pull request in the kata
```
Comment (klynnrif) [inline release/README.md]: I believe you want to capitalize "kata" for consistency (see line 33).

**Comment by jcvenegas** [comment]
@klynnrif thx fixed. Ready to merge.

**Comment by jcvenegas** [comment]
CI failing due to   https://github.com/kata-containers/tests/issues/485

**Comment by jcvenegas** [comment]
@klynnrif all passing now, just need your approval to merge this PR.

**Comment by chavafg** [comment]
lgtm

**Pull request merged**